### PR TITLE
Update websites.json: add a Nostr client

### PR DESCRIPTION
**Pull request by BlowaterNostr**
### Describe the changes you have made in this PR
Add https://blowater.deno.dev/ to Nostr website page

### Type of change
- `docs`: Documentation update

### CODE REVIEW

**Comment by BlowaterNostr** [comment]
@reneaaron Will reproduce and give you a better report.

**Comment by reneaaron** [comment]
@BlowaterNostr Cool, thanks! 🙌 Feel free to open a new issue for that 👍

**Comment by BlowaterNostr** [comment]
Maybe it's not necessarily a bug of Alby. But the way my application is interacting with it. The direct observation is that Using Alby on Firefox with Blowater will trigger a null reference exception very deep in Preact's code with no call stack from application code. With the same code, but Chrome works.

**Comment by reneaaron** [comment]
Okay. Is the code available publicly? Happy to take a quick look...

**Comment by BlowaterNostr** [comment]
@reneaaron the code is not open sourced yet. I will consider it once it reaches feature stability.

**Comment by BlowaterNostr** [comment]
@reneaaron I have found the reason and it's a good indicator that Alby should support concurrent API calls or at least throw errors when user concurrently calls.

The story is this, in Blowater, we concurrently call `encrypt/decrypt/signEvent/getPublicKey` in many different promise chains. The current behavior of `window.nostr` is to simply return the result of the first call in the current JS tick. For example, if
```
getPublicKey()
encrypt()
```
are called in the same tick. `await encrypt()` will return the public key. Although in the log Alby gives a warning message, but the code behavior is not a good design. I would categorize it as either a bug or a design flaw.

I worked around this problem by adding a queue in my application code to wait for the previous call to resolve before invoking the next.

But, because Blowater also use IndexedDB, therefore, wrong data before I added the queue were persisted. I happened to never delete the database on Firefox, so that only Firefox appears buggy. I deleted the database and everything works now.

**Comment by BlowaterNostr** [comment]
#2218 does not block the development of my client, can we merge this first? @reneaaron

**Comment by BlowaterNostr** [comment]
Since #2218 is just merged, can we merge this now?

File: src/app/screens/Discover/websites.json [hunk 1]
```diff
@@ -118,6 +118,12 @@
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/blockstack-logo.png",
         "url": "https://blogstack.io/"
       },
+      {
+        "title": "Blowater",
+        "subtitle": "A web chat client with delightful UX",
+        "logo": "https://blowater.deno.dev/logo.png",
```
Comment (reneaaron) [inline src/app/screens/Discover/websites.json:124]:
```suggestion
        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/blowater_thumbnail.png",
```

**Comment by BlowaterNostr** [comment]
@reneaaron 
image

Blowater isn’t showing up. Is this because the action failed?

**Comment by bumi** [comment]
@BlowaterNostr you might stil be on the old version. 
CleanShot 2023-05-07 at 11 42 38@2x

**Pull request merged**